api/v1/verticadb_types.go

```diff
@@ -416,12 +416,14 @@ const (
 	// leaving the secondary subclusters in read-only mode.  When the primary
 	// subcluster comes back up, we restart/remove all of the secondary
 	// subclusters to take them out of read-only mode.
-	OnlineUpgrade UpgradePolicyType = "Online"
-	// Like online upgrade, however it allows for writes. This is done by
+	// This is only used internally by the operator to differentiate between
```

Comment:
ReadOnlyOnlineUpgrade shouldn't be used internally.

Reply:
That's an old comment. Not relevant anymore, will update it